anvio/summarizer.py

```diff
@@ -1902,3 +1891,39 @@ def store_data_in_file(self, output_file_name_posfix, content):
         output_file_obj.write('%s\n' % content)
         output_file_obj.close()
 
+
+def benjamini_hochberg_procdure(p_values, fdr=0.05):
```

Comment:
so this doesn't do anything? ;)

By the way, this function (and `get_z_test_statistic`) would be much better if they were . in utils, in my opinion.

Reply:
Done :-)
https://github.com/merenlab/anvio/commit/c182b3ee6f4b5f6ba4ada33dfa2f76d6aee27cf7
https://github.com/merenlab/anvio/commit/a704917f417b104c5de9666c260c4b065cc1bddd

Thank you!